GRANT UPDATE ON LARGE OBJECT 1006 TO regressuser3;
NOTICE:  00000: test_event_trigger: ddl_command_start GRANT
ERROR:  42704: large object 1006 does not exist
